generation/emitter/remap.rsp

```diff
@@ -197,3 +197,205 @@ IAMMultiMediaStream::AddMediaStream::dwFlags=AMMSF_MS_FLAGS
 IMultiMediaStream::GetInformation::pdwFlags=MMSSF_GET_INFORMATION_FLAGS*
 IIdentityProvider::Advise::dwIdentityUpdateEvents=IdentityUpdateEvent
 IIdentityAdvise::IdentityUpdated::dwIdentityUpdateEvents=IdentityUpdateEvent
+LoadImageA:type=IMAGE
+CopyImage:type=IMAGE
+LoadImageW:type=IMAGE
+IMbnRadioEvents.OnSetSoftwareRadioStateComplete:status=IMbnRadioEvents.OnSetSoftwareRadioStateComplete_statusFlags
```

Comment:
@sotteson1 How am I supposed to encode method names that belong to an interface?